SuperRonanCraft/bootcamp-Tomodachi
# 48 import auth use mutation from apollo and login in user from mutation

## Purpose

This got a bit out of hand and I ended up doing a lot more than I should have in this branch. This added the ability to login and sign up, added a whole new page for signing up, added a new button to the nav, and added client side mutations to the project.

Fixes #48 

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

- path/to/file
  - client/package.json
  - client/src/components/LoginForm.jsx
  - client/src/components/SignUpForm.jsx
  - client/src/components/landing/nav.jsx
  - client/src/components/utils/auth.js
  - client/src/components/utils/mutations.js
  - client/src/main.jsx
  - client/src/pages/SignUp.jsx
  - client/src/pages/login.jsx
  - package.json
  - server/schemas/resolvers.js

## Steps to Reproduce or Test

- clone the branch
- npm i
- npm run dev
- go to localhost:3000
- Make sure nothing is broken :)
